imod/tests/test_mf6/test_multimodel/test_mf6_partitioning_unstructured.py

```diff
@@ -435,20 +360,22 @@ def test_partition_transport_multispecies(    tmp_path: Path,
     
     
     circle_model_transport_multispecies.write(tmp_path/"original")
-    new_circle_model = circle_model_transport_multispecies.split(partition_array)
-
     circle_model_transport_multispecies.run()
-    new_circle_model.write(tmp_path/"split")
-    new_circle_model.run()
-
     conc = circle_model_transport_multispecies.open_concentration()
     head = circle_model_transport_multispecies.open_head()
 
+
+    new_circle_model = circle_model_transport_multispecies.split(partition_array)
+    new_circle_model.write(tmp_path/"split")
+    new_circle_model.run()
     conc_new = new_circle_model.open_concentration()
     head_new = new_circle_model.open_head()
+    head_new = head_new.ugrid.reindex_like(head)   
+    conc_new = conc_new.ugrid.reindex_like(conc)
+
     
-    np.testing.assert_allclose(conc.values, conc_new["concentration"].values, rtol=1e-5, atol=1e-3)
-    np.testing.assert_allclose(head.values, head_new["head"].values, rtol=1e-5, atol=1e-3)
+    np.testing.assert_allclose(conc.values, conc_new["concentration"].values, rtol=4e-4, atol=5e-3)
+    np.testing.assert_allclose(head.values, head_new["head"].values, rtol=0.008, atol=0.15)
```

Comment:
I see the criteria are set at seemingly scary levels, however heads in the example range from 2000 to 0 m. I think it is good to add a comment that heads range from 2000 to 0 m (implying that we shouldn't worry about too much about it...).

Reply:
done. I now set the higher tolerances only for the concentric partition scheme, and added a comment.